common/src/registers/MachRegister.C

```diff
@@ -39,6 +39,12 @@ namespace Dyninst {
           return MachRegister(reg & 0xfffff0ff);
```

Comment:
clear the whole 16-bits of the subrange with a mask of `0xffff00ff`.

Reply:
Agreed. Fixed.